multitool/defs.bzl

```diff
@@ -0,0 +1,54 @@
+"""fork of rules_multitool
```

Comment:
Is this public API or just an internal util? Can we break it later being the real question...